tests/prague/eip7692_eof_v1/eip7069_extcall/test_gas.py

```diff
@@ -50,25 +57,29 @@ def gas_test(
 
     sender = pre.fund_eoa(10**18)
 
-    address_baseline = pre.deploy_contract(setup_code + tear_down_code)
-    address_subject = pre.deploy_contract(setup_code + subject_code + tear_down_code)
+    address_baseline = pre.deploy_contract(Container.Code(setup_code + tear_down_code))
+    address_subject = pre.deploy_contract(
+        Container.Code(setup_code + subject_code + tear_down_code)
+    )
+    # GAS, POP, CALL, 7 times PUSH1 - instructions charged for at every gas run
+    gas_single_gas_run = 2 + 2 + 100 + 7 * 3
     address_legacy_harness = pre.deploy_contract(
         code=(
             # warm subject and baseline without executing
             (Op.BALANCE(address_subject) + Op.POP + Op.BALANCE(address_baseline) + Op.POP)
-            # cold gas run
+            # Baseline gas run
             + (
                 Op.GAS
-                + Op.CALL(address=address_subject, gas=500_000)
+                + Op.CALL(address=address_baseline, gas=500_000)
```

Comment:
```suggestion
                + Op.CALL(address=address_baseline, gas=call_gas)
```
I think the call gas value should be a parameter (with a default) to this function.

Reply:
I have a different proposal, the `gas` for these 3 calls is irrelevant, and actually if not enough would interfere with the idea of the test - gas should never run out.

How about `Op.GAS` is sent here, this would require only minimal adjustments to the harness, would this make sense @shemnon ?

In the same vein I'd set Transaction's gas_limit to be the block's gas_limit, which is a huge quantity by default.

Reply:
see 8d233acd